content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -95,12 +95,22 @@ At this point, the command has two additional side effects. The command installs
 
 ## 3. Generate Prisma Client for the Data Proxy
 
-Next, generate Prisma Client for the Data Proxy with the `--data-proxy` flag. Later, you use the [Data Proxy](/data-platform/classic-projects/data-proxy) to connect to your database over HTTP.
+Next, generate Prisma Client for the Data Proxy with the `--no-engine` flag. Later, you use the [Data Proxy](/data-platform/classic-projects/data-proxy) to connect to your database over HTTP.
+
+```terminal
+deno run -A --unstable npm:prisma generate --no-engine
```

Comment:
Is this even necessary or will Deno just ignore the engine file on deploy?

Reply:
No, the engine will be in the resulting generated client and will be included in the deployed artifact. I don't think it can be effectively tree-shaken